pom.xml

```diff
@@ -17,15 +17,15 @@
         <spark.version.minor>1</spark.version.minor>
 
         <!-- other dependencies -->
-        <kusto-sdk.version>3.2.1</kusto-sdk.version>
+        <kusto-sdk.version>4.0.3</kusto-sdk.version>
         <specs2.version>3.6.5</specs2.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
-        <azure-storage.version>8.6.5</azure-storage.version>
+        <azure-storage.version>8.6.6</azure-storage.version>
```

Comment:
Is this storage consistent with the storage version we use in Java SDK ?

Reply:
Seems this is now moved. And the same is used in Java SDK. We can also move from [com.azure](https://mvnrepository.com/artifact/com.azure) » azure-storage-blob
![image](https://user-images.githubusercontent.com/102726227/223131952-f479df7a-07e1-4e85-b28e-e14eed5acbf1.png)

Reply:
Will open a new issue for this.

Reply:
sure, would it be possible to see what changes will be needed for this. May be try the update in this same PR ?

Reply:
Sure. There are some new classes introduced, I guess. Will check them out.

Reply:
Actually if this is the PR where we upgrade major version this has to be on this PR
java sdk removed all dependencies on the old sdk